apiExamples/basic.html

```diff
@@ -1 +1,7 @@
-<auro-breadcrumb>Hello World</auro-breadcrumb>
+<auro-nav>
+  <auro-nav-breadcrumb href="#">Home</auro-nav-breadcrumb>
+  <auro-nav-breadcrumb href="#">Level 2</auro-nav-breadcrumb>
+  <auro-nav-breadcrumb href="#">Level 3</auro-nav-breadcrumb>
+  <auro-nav-breadcrumb href="#">Level 4</auro-nav-breadcrumb>
+  <auro-nav-breadcrumb>Current</auro-nav-breadcrumb>
```

Comment:
The href causes the user to navigate to somewhere odd. let's make these go to the default aur-nav url on the docsite.
```suggestion
  <auro-nav-breadcrumb href="http://auro.alaskaair.com/components/auro/nav">Home</auro-nav-breadcrumb>
  <auro-nav-breadcrumb href="http://auro.alaskaair.com/components/auro/nav">Level 2</auro-nav-breadcrumb>
  <auro-nav-breadcrumb href="http://auro.alaskaair.com/components/auro/nav">Level 3</auro-nav-breadcrumb>
  <auro-nav-breadcrumb href="http://auro.alaskaair.com/components/auro/nav">Level 4</auro-nav-breadcrumb>
  <auro-nav-breadcrumb>Current</auro-nav-breadcrumb>
```